### DISCUSSION TITLE
Should Procedures be separate from Subject

### DISCUSSION THREAD

**Discussion by saskiad** [AllenNeuralDynamics/aind-data-schema#187] [General]
Should the content of the procedures - and particularly the "subject procedures" (e.g. surgeries) - be described in the Subject schema? These are all procedures done to the subject prior to data acquisition. Conceptually it might make sense to have them in the same file.

One argument against is that the information comes from different places - e.g. LabTracks vs SWR on Sharepoint.

**Comment by saskiad** [comment]
@dyf any thoughts on this? 

It touches on the specimen procedures as well - I'm wondering if it really makes sense to have subject and specimen procedures in the same place. Perhaps subject procedures could live in the subject schema and have a separate specimen procedure schema.

**Reply by dyf** [reply]
I think it makes sense to bring procedures into the subject schema.  It's where I would expect to find it. This would impact the metadata service work, so we should time this somewhat carefully.

I think this generalizes to the question of whether we should combine all of the files together.  Perhaps adding a top level "DataAsset" schema?

**Reply by dyf** [reply]
To the argument about data sources - honestly I'd rather abstract that away. Data comes from many places, and the reasons why don't always make sense. A critical function of the metadata service is honestly to hide that complexity from users.

**Comment by dyf** [comment]
@saskiad I think this is less important when we implement the `DataAsset`. Is that fair to say?

**Comment by saskiad** [comment]
yes.
